### PR TITLE
Swap Chameleon commenting templates with Jinja ones

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony @nathanbegbie ready for review. Tests will fail until I merge the testing branch.

@nathanbegbie I haven't entirely followed the fufu pattern, but I think this can be converted in springboard. Organisms: `comment_list` and `comment_form` macros; molecules: `comment` macro; ecosystem: `comment_page`. Related: I think we should have a file per "unit" or macro, placed in subfolders `molecules/`, `organisms/`, etc. That way we can override each one individually.

**Comment by Rizziepit** [comment]
@miltontony @nathanbegbie tests passing and ready for review

**Comment by miltontony** [comment]
:+1:

File: cms/templates/comments/comment_form.jinja2 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        {% if request.user %}
+        <form action="{{ request.route_url('comments', uuid=content_obj.uuid, _query=request.GET) }}" method="post">
+            {{ form_utils.errors(form.error) }}
+            {% for field in form %}{{ form_utils.field(field) }}{% endfor %}
+            <input name="submit" type="submit" value="{{ _('Add your comment') }}" class="btn btn-wide" />
+        </form>
+        {% else %}
+        <p>
+            {{ _('You have to be signed in to add a comment') }}. <a href="{{ request.route_url('redirect_to_login') }}">{{ _('Sign in here') }}</a>.
```
Comment (nathanbegbie) [inline cms/templates/comments/comment_form.jinja2]: @Rizziepit Is this the translate syntax?

Comment (Rizziepit) [inline cms/templates/comments/comment_form.jinja2]: Both `_` function and `{% trans %}...{% endtrans %}` work.

**Comment by nathanbegbie** [comment]
Cool :) Thanks :+1:

**Pull request merged**